Formula/t/three-body.rb

```diff
@@ -0,0 +1,18 @@
+class ThreeBody < Formula
+  desc "三体编程语言 Three Body Language written in Rust"
+  homepage "https://github.com/rustq/3body-lang"
+  url "https://github.com/rustq/3body-lang/releases/download/0.3.0-tap.2/3body-lang.tar.gz"
+  sha256 "b40cb3beaa6c40b610c05385792f40eddb19e95abeb6e1863702641d7f7e9c8b"
+  license "MIT"
+  head "https://github.com/rustq/3body-lang.git", branch: "main"
```

Comment:
if the default branch is not main (which is the case right now, you will need to change it in the repo settings)

![image](https://github.com/Homebrew/homebrew-core/assets/1580956/b9a79f78-7769-4b33-91b1-4c3bed420853)

Reply:
we have an audit check for that.

Reply:
Ok thanks, I will merge the branch into main and then make it correct.

Reply:
👍